src/core/api/index.js

```diff
@@ -8,9 +9,14 @@ import { schema as normalizrSchema, normalize } from 'normalizr';
 import { oneLine } from 'common-tags';
 import config from 'config';
 
+import { initialApiState } from 'core/reducers/api';
 import { ADDON_TYPE_THEME } from 'core/constants';
 import log from 'core/logger';
 import { convertFiltersToQueryParams } from 'core/searchUtils';
+/* eslint-disable no-duplicate-imports */
```

Comment:
No way to import types separately, like `import { ErrorHandler }, type { ErrorHandlerType } from 'core/errorHandler';`?